### PR TITLE
Fixed missing handling of nodata for count aggregator with column

### PR DESCRIPTION
**Pull request by jlstevens**
The `count_cat` aggregators were handled but `count` wasn't. Simple fix that shouldn't need much discussion once a corresponding unit test is added.

### CODE REVIEW

**Review by jbednar — APPROVED** [review]

**Review by jbednar — APPROVED** [review]
Thanks!

**Comment by jlstevens** [comment]
The test failure for 3.6 ubuntu seems unrelated to this PR. I tried restarting the tests but the same failure reappeared:

```
self = <holoviews.plotting.bokeh.callbacks.PointDrawCallback object at 0x7f197416c908>
plot_id = None

    def initialize(self, plot_id=None):
        plot = self.plot
        stream = self.streams[0]
>       cds = plot.handles['source']
E       KeyError: 'source'
```

@philippjfr Any idea what this failure is about? Is it simply a transient that happened twice in a row?

At any rate, happy to see this PR merged now.

File: holoviews/operation/datashader.py [hunk 1]
```diff
@@ -297,8 +297,11 @@ def _get_agg_params(self, element, x, y, agg_fn, bounds):
         params = dict(get_param_values(element), kdims=[x, y],
                       datatype=['xarray'], bounds=bounds)
 
-        kdim_list = '_'.join(str(kd) for kd in params['kdims'])
-        vdim_prefix = self.vdim_prefix.format(kdims=kdim_list)
+        if self.vdim_prefix is not None:
```
Comment (philippjfr) [inline holoviews/operation/datashader.py:300]: Not real different but no point doing the kdims join if there is no prefix string:

```suggestion
        if self.vdim_prefix:
```

Comment (jlstevens) [reply holoviews/operation/datashader.py:300]: Sure!

**Pull request merged**